orc8r/cloud/helm/orc8r/charts/metrics/templates/alertmanager.service.yaml

```diff
@@ -8,7 +8,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ .Release.Name }}-alertmanager
+  name: alertmanager
```

Comment:
revert the name changes - we use a static config file to route to these services, so the orc8r- prefix is actually important (https://facebookincubator.github.io/magma/docs/orc8r/deploy_install#static-configuration-files)

Reply:
Thanks, Jacky. revered the changes.